armada/handlers/k8s.py

```diff
@@ -58,3 +58,10 @@ def get_namespace_pod(self, namespace="default"):
         '''
 
         return self.client.list_namespaced_pod(namespace)
+
+    def get_all_pods(self):
+        '''
+        This will return a list of pods across all namespaces
+        '''
+
+        return self.client.list_pod_for_all_namespaces()
```

Comment:
this begs a rather big question - if you are just creating a method to call another method - why not cut out the middle man?